### ISSUE TITLE
[a11y audit] Testing: MHV Prescription Skill 4.2.17: Communicate privacy policies

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#67380]
## 4.2.17 Communicate privacy policies
- **Testing Status:** `Completed`
- **Findings**: `Present`
- **Audit Spreadsheet:** [Link to 4.2.17](https://docs.google.com/spreadsheets/d/12NgigOQTKgsSexnq_ioUwzXQLtRGRcrIYN--yb3i2Ig/edit#gid=15026979877)

### Testing Details
- **URL**: `dev`
- **Credentials**: user `#228`
- **OS**: MacOS 13.6
- **Browser**: Chrome, rolling release
- **a11y Audit Finding,** related to [4.2.17 Communicate privacy policies](https://mitre.github.io/chatbot-accessibility-playbook/docs/4_2_17.html)
- **Relates to:** COGA only [COGA 4.5.11 Help the user stay safe](https://www.w3.org/TR/coga-usable/#help-the-user-stay-safe-pattern)
- **VPAT Status:** Not evaluated
- **Impacts**: Cognitive
- **Experience Standards:** n/a
- **Severity Rubric**: n/a
- **[Update reflected in Audit Spreadsheet](https://docs.google.com/spreadsheets/d/12NgigOQTKgsSexnq_ioUwzXQLtRGRcrIYN--yb3i2Ig/edit#gid=1502697987)** ✅

I tested for the following:

> Users should understand what data must be saved to complete the task, what privacy policies protect that data, and what degree of privacy users would expect with the data.

### Does Not Pass 🔴 

The static page containing the chatbot features great information about data usage:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/b266f602-8c32-4e9c-907a-6ced28f6f15f)

But the chatbot **itself** doesn't have this information. It only has this warning before you click "Start chat:"

> Please don’t type any personal information such as your name, address, or anything else that can be used to identify you.

It's very easy to miss the "What information we collect when you use the chatbot" accordion. And if this chatbot is ever inserted onto a different static page, the user wouldn't have access to this information at all.

### Recreating the Issue 🖥️ 

To view the issue, go to the prescription skill: https://dev.va.gov/contact-us/virtual-agent/

Note where the privacy information is located on the static page.

### How to Fix 🛠️ 

Add copy to the chatbot beyond `Please don’t type any personal information such as your name, address, or anything else that can be used to identify you.` Some text that gives the basics of the privacy policy, then links to the policy itself. 

We recommend that you work with the CAIA content team to create this language.
